Source/UI/Authentication/Process/AuthenticationModels.swift

```diff
@@ -50,10 +50,12 @@ public enum Authentication {
         }
         
         public struct CommonErrors {
-			public let wrongPin: String
-			public let wrongPassword: String
+            public let wrongPin: String
+            public let wrongPassword: String
+            public let biometryNotRecognized_TouchId: String
+            public let biometryNotRecognized_FaceId: String
 			public let activationWasRemoved: String
```

Comment:
Just fix also this remaining tabs :D That's due to my Xcode, which randomly selects tabs or spaces when I'm indenting the code :)

Reply:
Fixed :)